src/raftstore/store/fsm/apply.rs

```diff
@@ -379,12 +381,15 @@ impl ApplyContext {
         if self.kv_wb.as_ref().map_or(false, |wb| !wb.is_empty()) {
             let mut write_opts = WriteOptions::new();
             write_opts.set_sync(need_sync);
+            let mut perf_task = PerfTask::new(APPLY_ROCKSDB_PERF_CONTEXT_HISTOGRAM_VEC.local(), PerfLevel::EnableTime, "raft");
+            perf_task.start_perf();
             self.engines
                 .kv
                 .write_opt(self.kv_wb(), &write_opts)
                 .unwrap_or_else(|e| {
                     panic!("failed to write to engine: {:?}", e);
                 });
+            drop(perf_task);
```

Comment:
we don't need to call it explicitly

Reply:
create a scope for the perf task instead of calling drop? like
```
{
    let mut perf_task = PerfTask::new(...); // implicitly call perf_task.start_perf().
    // do some work.
}
```